src/messaging/messaging-types.ts

```diff
@@ -59,6 +59,7 @@ export interface MulticastMessage extends BaseMessage {
 export interface Notification {
   title?: string;
   body?: string;
+  image?: string;
```

Comment:
We called these `imageUrl` in Java. Might want to do the same here.

Reply:
@hiranya911 
Do you think it is sufficient to change just the public API `index.d.ts` here or should we use the same parameter name `imageUrl` everywhere in our code?